.gitignore

```diff
@@ -1,4 +1,3 @@
-app/config/custom.json
```

Comment:
custom.json shouldn't be included in the public repo, otherwise it will affect the public site as well